### DISCUSSION TITLE
Build issue, error and only compiling the JS files. CSS and Images not copying to /dist

### DISCUSSION THREAD

**Discussion by mick9000** [nicolas-t/Chocolat#135] [Q&A] [answered]
I have a contribution to this project, but I face an issue when running "npm run dev".
I'm fairly new to the builders and I'm not sure why the JS files get processed, but images and CSS don't.
I get the following error when trying to run the command. Can anyone advise how to build?

![image](https://user-images.githubusercontent.com/85680866/157447335-93a6f5c6-3673-4ed1-b65c-a9311d39fdf7.png)

rollup config that I did not modify
![image](https://user-images.githubusercontent.com/85680866/157447570-029cea51-106a-4f38-a9c6-5693df64753d.png)

package.json which I did not modify
![image](https://user-images.githubusercontent.com/85680866/157447665-96b2a118-8fa4-4f21-910d-304a4be1b5a6.png)

**Comment by nicolas-t** [comment, marked as answer]
Hello,

You are using rollup 2.70 whereas chocolat is using rollup 0.68.2.
If you run `npm install` it should install the correct version

**Reply by mick9000** [reply]
This worked. I'll commit this evening. Thanks very much
